code/datums/materials/alloys.dm

```diff
@@ -0,0 +1,124 @@
+/** Materials made from other materials.
+  */
+/datum/material/alloy
+	name = "alloy"
+	desc = ""
+	/// The materials this alloy is made from weighted by their ratios.
+	var/list/composition = null
+	/// Breakdown flags required to reduce this alloy to its component materials.
+	var/req_breakdown_flags = BREAKDOWN_ALLOYS
+
+/datum/material/alloy/return_composition(amount=1, breakdown_flags)
+	if(req_breakdown_flags & !(breakdown_flags & req_breakdown_flags))
+		return ..()
+
+	. = list()
+	var/list/cached_comp = composition
+	for(var/comp_mat in cached_comp)
+		var/datum/material/component_material = SSmaterials.GetMaterialRef(comp_mat)
+		var/list/component_composition = component_material.return_composition(cached_comp[comp_mat], breakdown_flags)
+		for(var/comp_comp_mat in component_composition)
+			.[comp_comp_mat] += component_composition[comp_comp_mat] * amount
+
+
+/** Plasteel
+  *
+  * An alloy of iron and plasma.
+  */
+/datum/material/alloy/plasteel
+	name = "plasteel"
+	desc = "The heavy duty result of infusing iron with plasma."
+	color = "#828282"
+	value_per_unit = 0.135
+	strength_modifier = 1.5
```

Comment:
force 21 mace that can break down normal doors go brrrrrrr

(tbh, a force 21 weapon isn't too out of line with other space loot, but the problem here is that cleric maces themselves are too weak, not that other materials are too weak, and plasteel eclipses those other materials in power level by a significant amount. nerf these alloys so that we can buff cleric maces and knight armor.)

Reply:
Plasteel's strangth modifier has been to parity with titanium. Is this sufficient?